mne_qt_browser/_pg_figure.py

```diff
@@ -1059,7 +1059,14 @@ def mousePressEvent(self, event):
 
     def mouseMoveEvent(self, event):
         """Customize mouse move events."""
-        self._set_range_from_pos(event.pos())
+        # This temporarily circumvents a bug, which only appears on windows
+        # and when pyqt>=5.14.2 is installed from conda-forge.
+        # It leads to receiving mouseMoveEvents all the time when the Mouse
+        # is moved through the OverviewBar, even when now MouseBUtton is
+        # pressed. Dragging the mouse on OverviewBar is then
+        # not possible anymore.
+        if not sys.platform.startswith("win32"):
```

Comment:
My impression (by reading stack overflow etc) is that the preferred and most reliable way for platform detection these days is via `platform.system()`, which would return `"Windows"` in Windows systems

Reply:
I see, thank you!